templates/30-validation-webhook.servicemonitor.yaml.tmpl

```diff
@@ -0,0 +1,17 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    name: "#SVCNAME#"
+  name: "#SVCNAME#"
+  namespace: "#NAMESPACE#"
+spec:
+  endpoints:
```

Comment:
This servicemonitor is for an /https endpoint, but according to `metrics.py` the endpoint is `/metrics`